var/spack/repos/builtin/packages/umt/package.py

```diff
@@ -0,0 +1,196 @@
+                    entries.append(cmake_cache_string("TETON_OpenMP_Offload_Fortran_FLAGS", "-qoffload -qtgtarch=sm_70"))
+            else:
+                entries.append(cmake_cache_string("TETON_OpenMP_Fortran_FLAGS_RELEASE", "-fopenmp"))
+                entries.append(cmake_cache_string("TETON_OpenMP_Fortran_FLAGS_DEBUG", "-fopenmp"))
+                entries.append(cmake_cache_string("TETON_OpenMP_CXX_FLAGS_RELEASE", "-fopenmp"))
+                entries.append(cmake_cache_string("TETON_OpenMP_CXX_FLAGS_DEBUG", "-fopenmp"))
+
+        if spec.satisfies("%cce"):
+            entries.append(cmake_cache_option("STRICT_FPP_MODE", True))
+            entries.append(cmake_cache_string("TETON_OpenMP_Offload_LINK_FLAGS", "-ldl"))
+
+        return entries
+
+    def initconfig_hardware_entries(self):
+        spec = self.spec
+        entries = super(Umt, self).initconfig_hardware_entries()
+
+        #######################
+        # Parallelism
+        #######################
+        if "+cuda" in spec:
+            entries.append(cmake_cache_option("ENABLE_CUDA", True))
+        if "+openmp" in spec:
+            entries.append(cmake_cache_option("ENABLE_OPENMP", True))
+        if "+openmp_offload" in spec:
+            entries.append(cmake_cache_option("ENABLE_OPENMP_OFFLOAD", True))
+
+        if spec.satisfies("%cce"):
+            entries.append(cmake_cache_option("OPENMP_HAS_USE_DEVICE_ADDR", True))
+            entries.append(cmake_cache_option("OPENMP_HAS_FORTRAN_INTERFACE", True))
+
+        return entries
+
+    def initconfig_mpi_entries(self):
+        entries = super(Umt, self).initconfig_mpi_entries()
+
+        return entries
+
+    def initconfig_package_entries(self):
+        spec = self.spec
+        entries = []
+
+        #######################
+        # Disable features not needed by UMT
+        #######################
+        entries.append(cmake_cache_option("ENABLE_MINIAPP_BUILD", True))
+        entries.append(cmake_cache_option("ENABLE_TESTS", True))
+        entries.append(cmake_cache_option("ENABLE_SILO", False))
+        entries.append(cmake_cache_option("ENABLE_HDF5", False))
+
+        entries.append(cmake_cache_path("CONDUIT_ROOT", spec["conduit"].prefix))
+
+        if "+mfem" in spec:
+            entries.append(cmake_cache_option("ENABLE_MFEM", True))
+            entries.append(cmake_cache_path("MFEM_ROOT", spec["mfem"].prefix))
+            if "hypre" in spec:
+                entries.append(cmake_cache_path("HYPRE_ROOT", spec["hypre"].prefix))
+            if "metis" in spec:
+                entries.append(cmake_cache_path("METIS_ROOT", spec["metis"].prefix))
+            
+        if "+caliper" in spec:
+            entries.append(cmake_cache_option("ENABLE_CALIPER", True))
+            entries.append(cmake_cache_path("CALIPER_ROOT", spec["caliper"].prefix))
+            if "adiak" in spec:
+                entries.append(cmake_cache_path("ADIAK_ROOT", spec["adiak"].prefix))
```

Comment:
```suggestion
            entries.append(cmake_cache_path("ADIAK_ROOT", spec["adiak"].prefix))
```
The dependency guarantees it's there